providers/evercrypt/fst/EverCrypt.Hash.fst

```diff
@@ -413,7 +413,7 @@ let update_multi #a s prevlen blocks len =
 // Re-using the higher-order stateful combinator to get an instance of
 // update_last that is capable of calling Vale under the hood
```

Comment:
this comment is outdated and the function itself can probably go since it no longer serves its original purpose